src/components/look-controls.js

```diff
@@ -147,11 +147,19 @@ module.exports.Component = registerComponent('look-controls', {
     hmdEuler.setFromQuaternion(hmdQuaternion, 'YXZ');
     if (isMobile) {
       // In mobile we allow camera rotation with touch events and sensors
-      rotation = {
-        x: radToDeg(hmdEuler.x) + radToDeg(pitchObject.rotation.x),
-        y: radToDeg(hmdEuler.y) + radToDeg(yawObject.rotation.y),
-        z: radToDeg(hmdEuler.z)
-      };
+      if (this.data.reverseMouseDrag) {
```

Comment:
Well. It depends on the user and the content. Not sure it that's a fair assumption. You might want to also let the developer make the choice to have consistency across desktop / mobile.